### DISCUSSION TITLE
When convert cp2k data to deepmd input file, there is a circular import error.

### DISCUSSION THREAD

**Discussion by lyy991127** [deepmodeling/dpdata#764] [Q&A] [answered]
Hello, I received the following error when converting the CP2K AIMD output file to a DeepMd input file. I am using dpdata version 0.2.17. and cp2kdata version 0.7.1. 
My code
import dpdata
cp2kmd_dir = "1100"
cp2kmd_output_name = "output.log"
dp = dpdata.LabeledSystem(cp2kmd_dir, cp2k_output_name=cp2kmd_output_name, fmt="cp2kdata/md")
print(dp)
Error
![1733390608292](https://github.com/user-attachments/assets/f23a11bd-1e82-464b-be58-bd562bf56b0c)

**Comment by robinzyb** [comment]
it seems you have a circular import. could you use virtual python environment to execute the script?

**Reply by lyy991127** [reply]
Thanks for the reply, my problem has been soved.

**Comment by njzjz** [comment]
You meet this: https://discuss.python.org/t/warning-when-importing-a-local-module-with-the-same-name-as-a-2nd-or-3rd-party-module/27799

**Reply by lyy991127** [reply, marked as answer]
Thank you for your reply, there is no cicular import error now.
